optuna/integration/allennlp.py

```diff
@@ -56,8 +55,11 @@ class AllenNLPExecutor(object):
     The interface may change without prior notice to correspond to the update.
 
     See the examples of `objective function <https://github.com/optuna/optuna/blob/
-    master/examples/allennlp/allennlp_jsonnet.py>`_ and
-    `config file <https://github.com/optuna/optuna/blob/master/
+    master/examples/allennlp/allennlp_jsonnet.py>`_.
+
+    From Optuna v2.1.0, user has to cast their parameters by using methods in Jsonnet.
+    Call ``std.parseInt`` for integer, or ``std.parseJson`` for floating point.
+    Please see the example of `config file <https://github.com/optuna/optuna/blob/master/
     examples/allennlp/classifier.jsonnet>`_.
```

Comment:
I think it's better to make these lines as "Note".
```suggestion
    .. note::
        From Optuna v2.1.0, user has to cast their parameters by using methods in Jsonnet.
        Call ``std.parseInt`` for integer, or ``std.parseJson`` for floating point.
        Please see the example of `config file <https://github.com/optuna/optuna/blob/master/
        examples/allennlp/classifier.jsonnet>`_.
```